.github/workflows/e2e-next-example.yml

```
@@ -14,7 +14,7 @@ jobs:
       PR_BRANCH: ${{github.event.pull_request.head.ref}}
     strategy:
       matrix:
-        node: ['16', '18', '20']
+        node: '18', '20']
```

It looks like all these need the open `[`

```suggestion
        node: ['18', '20']
```